cc-visionary/StickerMe
# [US7 & US10] Customer Edit Sticker Functionality and Save that Character to the Database

## Tasks Done
- Save the edited characters to the database
- Save the character details such as their poses, name, and description
- Add contacts that was inputted to the databases
- Form Validation for Order Form
- Add order to the database

## To test:
1. Click the `Characters` in the NavBar
2. Click on the `Create a new Character` button
3. Customize your character, select poses, set name and description, then add contact details.
4. Once you go back to Characters page, you'll see the existing character there (only the data for now since there isn't any design for it yet)
